src/themes/default/plugin.js

```diff
@@ -58,7 +58,8 @@ PluginRegistry.add('themes', {
             'primary-color-lightness': '60%',
             'secondary-color-saturation': '70%',
             'color-warning-light': 'hsla(var(--primary-color-hsl), 0.15)',
-            'logo-color': 'var(--primary-color-dark)'
+            'logo-color': 'var(--primary-color-dark)',
+            'keyboard-shortcuts': 'var(--secondary-color)'
```

Comment:
Why did this was added here? Can't we just use the secondary-color var in the dark theme stylesheet?

Reply:
```
.hotkeys-text, .keyboard-shortcut-description-item-description {
    color: var(--secondary-color);
}
```
Adding only this to the bootstrap-graphdb-theme-dark-auto.css applies the color in light theme as well.